project-reports/2023/2023-Q1-Hyperledger-Iroha.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: default
+parent: 2023
+grand_parent: Project Updates
+---
+
+# Project Health
+
+Currently, the active maintenance of Iroha 1 has stopped, and there are no plans for major releases. There are discussions on making the release for the current RC, but this is undecided because it’ll take more than one team to switch from their everyday tasks.
```

Comment:
```suggestion
Currently, the active maintenance of Iroha 1 has stopped, and there are no plans for major releases. There are discussions on making releasing the current RC, but this is undecided because it will require more than one team to switch from their everyday tasks to focus on this.
```

Reply:
`on making releasing` : maybe just `on releasing`?

Reply:
I agree with @outoftardis, even though I am not a native speaker

Reply:
I believe @WRRicht3r just forgot to delete a word there